website/docs/rbs-support.md

```diff
@@ -624,13 +624,6 @@ specify them using `@` annotation comments.
 The following signatures are equivalent:
 
 ```ruby
-# @abstract
-#: (Integer) -> void
-def foo1(x); end
-
-sig { abstract.params(x: Integer).void }
-def foo2(x); end
-
```

Comment:
wdyt keeping this example with another annotation like `@override`?

Reply:
There is one just below ⬇️ Or let me know if you were thinking about something else

Reply:
I'm blind 🤦